src/write/string.rs

```diff
@@ -1,5 +1,9 @@
-use indexmap::IndexSet;
-use std::vec::Vec;
+use alloc::vec::Vec;
+
+#[cfg(feature = "std")]
+type IndexSet<K> = indexmap::IndexSet<K>;
+#[cfg(not(feature = "std"))]
+type IndexSet<K> = indexmap::IndexSet<K, hashbrown::hash_map::DefaultHashBuilder>;
```

Comment:
Is there any advantage to using `hashbrown` with the `ahash` feature instead of using `ahash` directly?

Reply:
Oh, that's a good idea! No advantage to using `hashbrown` over `ahash` as far as I can tell.

Reply:
Ah, well actually, the advantage would just be to simplify the dependency set, as we still need `hashbrown` to provide a `HashMap` implementation used in `/src/write/mod.rs`.

Thus, if I changed this to `ahash::RandomState`, we'd need to add another optional dependency on `ahash` to the `Cargo.toml` file, which is a tiny bit more complicated than having only `hashbrown`.